test/tracing/integrations/nativeframes.test.ts

```diff
@@ -216,6 +229,17 @@ describe('NativeFramesInstrumentation', () => {
   });
 
   it('does not set measurements on a transaction event for which finishFrames times out.', async () => {
+    asyncProcessorBeforeNativeFrames = async (event: Event) => {
+      await Promise.resolve();
+      await Promise.resolve();
+      await Promise.resolve();
+
+      global.Date.now = jest.fn(() => mockDate.getTime() + 2100); // hardcoded final frames timeout 2000ms
```

Comment:
```suggestion
      global.Date.now = jest.fn(() => mockDate.getTime() + 2100); // hardcoded final frames timeout 2100ms
```

Reply:
The timeout is 2000, that's why we move by 2100 to ensure it was triggered.